addons/website_slides/tests/test_slide_channel.py

```diff
@@ -370,3 +370,38 @@ def test_channel_enroll_policy(self):
 
         copied_channel = channel.copy()
         self.assertEqual(copied_channel.enroll, 'invite', "Copied channel should have the same enroll field value")
+
+    def test_invite_email_translation(self):
+        "Make sure that invitation emails are translated if unchanged when adding attendees to a course"
+        fr = self.env['res.lang']._activate_lang('fr_FR')
+        lang_install = self.env['base.language.install'].create({
+            'lang_ids': [fr.id],
+        })
+        lang_install.lang_install()   # make it so French is actually installed
+        jean = self.env['res.partner'].create({'name': 'Jean', 'lang': 'fr_FR'})
+
+        template = self.env.ref('website_slides.mail_template_slide_channel_enroll')
+        channel = self.env['slide.channel'].create({'name': 'Test Course'})
+        slide_channel_jean = self.env['slide.channel.partner'].create({
+            'channel_id': channel.id,
+            'partner_id': jean.id,
+        })
+
+        wizard = self.env['slide.channel.invite'].create({
+            'send_email': True,
+            'partner_ids': [jean.id],
+            'channel_id': channel.id,
+            'template_id': template.id,
+        })
+
+        mail_vals = wizard._prepare_mail_values(slide_channel_jean)
+        self.assertEqual(
+            mail_vals['body_html'],
+            wizard._render_field('body', [slide_channel_jean.id], set_lang=jean.lang)[slide_channel_jean.id],
```

Comment:
can you replace this one and the one below with the actual french content? so we don't reuse `_render_field` inside the test

Reply:
Indeed we should check the returned result directly, here it is a bit indirect. Note that you can create a small template and its translation in the test, so that we don't rely on existing data.